examples/pylab_examples/image_origin.py

```diff
@@ -1,25 +1,25 @@
-#!/usr/bin/env python
 """
 You can specify whether images should be plotted with the array origin
 x[0,0] in the upper left or upper right by using the origin parameter.
 You can also control the default be setting image.origin in your
 matplotlibrc file; see http://matplotlib.org/matplotlibrc
 """
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
 
-x = arange(100.0)
+x = np.arange(100.0)
 x.shape = (10, 10)
 
 interp = 'bilinear'
-#interp = 'nearest'
+# interp = 'nearest'
 lim = -2, 11, -2, 6
-subplot(211, axisbg='g')
-title('blue should be up')
-imshow(x, origin='upper', interpolation=interp)
-#axis(lim)
+plt.subplot(211, axisbg='g')
+plt.title('blue should be up')
+plt.imshow(x, origin='upper', interpolation=interp)
```

Comment:
I have suggest a 'jet jar' at work.  Every time someone sees jet on your
screen you have to put a $1 in.

On Tue, Aug 18, 2015 at 10:31 AM Ryan May <notifications@github.com> wrote:

> In examples/pylab_examples/image_origin.py
> <https://github.com/matplotlib/matplotlib/pull/4950#discussion_r37304755>:
>
> >  lim = -2, 11, -2, 6
> > -subplot(211, axisbg='g')
> > -title('blue should be up')
> > -imshow(x, origin='upper', interpolation=interp)
> > -#axis(lim)
> > +plt.subplot(211, axisbg='g')
> > +plt.title('blue should be up')
> > +plt.imshow(x, origin='upper', interpolation=interp)
>
> Adding cmap='jet' sounds like something your computer should shock you
> for doing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4950/files#r37304755>.
>